pkg/scheduler/algorithmprovider/defaults/compatibility_test.go

```diff
@@ -453,6 +484,17 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 					{Name: "InterPodAffinityPriority", Weight: 2},
 					{Name: "MostRequestedPriority", Weight: 2},
 				},
+				ExtenderConfigs: []schedulerapi.ExtenderConfig{{
+					URLPrefix:        "/prefix",
+					FilterVerb:       "filter",
+					PrioritizeVerb:   "prioritize",
+					Weight:           1,
+					BindVerb:         "bind", // 1.8 became case-insensitive and tolerated "bindVerb"
+					EnableHTTPS:      true,
+					TLSConfig:        &restclient.TLSClientConfig{Insecure: true},
+					HTTPTimeout:      1,
+					NodeCacheCapable: true,
```

Comment:
Is this test expect to include ALL fields in `ExtenderConfig` (since still some missing)? Or we just focus on `BindVerb` in this fix cycle.

Reply:
it includes all fields in extender config in the release being tested (1.8, in this case). I'll update the tests for 1.10 and 1.11 to include the fields added in those releases

Reply:
fixed, and reopened in https://github.com/kubernetes/kubernetes/pull/65424 (accidentally opened from my fork's master branch in this PR)